extensions/sql-database-projects/src/models/project.ts

```diff
@@ -129,14 +129,48 @@ export class Project implements ISqlProject {
 			console.error(utils.getErrorMessage(e));
 		}
 
-		// find all folders and files to include
-		for (let ig = 0; ig < this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.ItemGroup).length; ig++) {
-			const itemGroup = this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.ItemGroup)[ig];
+		// glob style getting files and folders for new msbuild sdk style projects
+		if (this._isMsbuildSdkStyleProject) {
```

Comment:
Making sure I'm understanding the flow here correctly:

1. Add all folders and sql files (for new-style projects)
2. Add explicitly-included individual sql files (both new and old projects)
3. Add explicitly-included globs (new only)
4. Remove excluded files (to do)
5. Remove Properties folder

I think this logic can be reorganized a bit to make it a little more streamlined, maybe by compiling a `Set()` of normalized includes, then excluding the excludes, then adding the results to the project list?

Reply:
yup that's correct. That's a good idea. I was planning on going back and cleaning up in a later PR since `readProjFile()` is getting a bit big and messy, so I'll look into streamlining the logic then, too.